modules/standard/IO.chpl

```diff
@@ -1991,19 +1998,22 @@ The temporary file will be created in an OS-dependent temporary directory,
 for example "/tmp" is the typical location. The temporary file will be
 deleted upon closing.
 
-Temporary files are always opened with :type:`iomode` ``iomode.cwr``;
-that is, a new file is created that supports both writing and reading.
-
-.. TODO:
-  Temporary files are always opened with :type:`iomode` ``iomode.cwrx``;
-  that is, a new file is created that supports both writing and reading.
+Temporary files are opened with :type:`iomode` ``iomode.cwr``; that is, a new
+file is created that supports both writing and reading.  When possible, it may
+be opened such that it can never be reached via any pathname and in such a way
+that it will avoid symlink attacks, but this is system dependent.
```

Comment:
I'm not so sure it won't be reachable by some pathname. Also it might be better to word it in a different way to focus on the OS support more than the problems solved. E.g.

> When possible, it may be opened using OS support for temporary files in order to make sure that a new file is created only for use by the current application.

Or something. In particular, the mention about symlink attacks felt a bit out of place to me here.